.gitignore

```
@@ -32,6 +32,7 @@ src/mixxx.res
 !/res/keyboard/*.cfg
 
 cache
+tools/__pycache__
```

```suggestion
__pycache__
```
Then these will be ignored regardless of location. I didn't notice this issue because I have it in my global gitignore.